### ISSUE TITLE
Feature request: keypad & page offset

### ISSUE THREAD

**Issue by Ram-222** [mb1986/rm-hacks#320]
Other than reading mode, the only feature I miss from ddvk's hacks is the toolbar keypad to quickly go to a specific document page. Whereas the current implementation just seems to open up the navigation bar at the bottom.

In particular, it allowed for a page offset to be accounted for such that the user could match up the pdf document page number to the source document numbering. This worked really smoothly and helped a lot for quick navigation of pdfs without hyperlinked contents.

Perhaps not quite as useful in notebooks.

Is this possible?

Thank you

**Comment by PepikVaio** [comment]
It's possible, it's already been addressed - the feature is in reMarkable, just put it on the panel :)

**Comment by Ram-222** [comment]
> It's possible, it's already been addressed - the feature is in reMarkable, just put it on the panel :)

Oh really, where? I can see the "go-to" page in page overview, but not an offset or quick way to go straight to a page without that view or dragging the bottom navigation bar. Sorry if I've missed it and requested something unnecessarily

**Comment by PepikVaio** [comment]
Here you are:

![image](https://github.com/mb1986/rm-hacks/assets/36114753/5a90db66-0a8b-4c8f-a370-0ff801d915c5)

![image](https://github.com/mb1986/rm-hacks/assets/36114753/188d9a58-24b4-4297-9413-dfa443583d7a)

![image](https://github.com/mb1986/rm-hacks/assets/36114753/07cb41c5-bad1-4d9d-bc42-617ed19b9cef)

**Comment by Ram-222** [comment]
Thanks! I see that but it's slower and the idea is if be in the comments page and want to just type in the page immediately.

Also the ability to offset to match the outrage numbers is key. Here's what the ddvk one did - at the top could type in the "actual" page number in the doc and it would apply the offset. E.g. if pdf was page 10 but the doc itself showed page 6 (after cover, contents and any i-vii etc pages), it would offset by 4 pages. So if you then typed in page 100 it would take you to page 100 of the textbook rather than page 104. I'll try and add image:

**Comment by Ram-222** [comment]
![Screenshot_20240405-101509](https://github.com/mb1986/rm-hacks/assets/165558687/e310214c-abd7-4fcb-93fb-4afc2ccf9057)

**Comment by Ram-222** [comment]
Here's a video/gif showing it in Action: https://www.reddit.com/r/RemarkableTablet/s/pVD2Eb5qkA

**Comment by PepikVaio** [comment]
Yes, I know what DDVK had, I was just pointing out that the "Go to page" feature is already in reMarkable - just put it on the panel. 

Thanks for explaining what the shift is good for - I didn't know that.
